### ISSUE TITLE
The training loss is much smaller than the provided log

### ISSUE THREAD

**Issue by CuttlefishXuan** [open-mmlab/mmsegmentation#355]
Hi, when I trained the Fast-SCNN model, I get the initial loss is about 0.5. And for training 160k iterations, the loss decreased from 0.5 to 0.04, and the eval miou is about 0.60.
According to the provided log file (https://github.com/open-mmlab/mmsegmentation/tree/master/configs/fastscnn), the initial loss is about 1, and 0.5 for 80k iterations. And the eval miou is 0.69
So I am confused about the loss function, does the both training process use the default nn.CrossEntropyLoss() with size_average=True? 

For my settings, batch size = 2 gpus x 8 imgs = 16, initial learning rate = 0.06

**Comment by Ashsur** [comment]
I have the same question.
I`m training a binary classifier model(foreground/background) based on Fast-SCNN, I get the initial loss is about 0.03, and the loss decreased to 0.003 after 100k iter.

**Comment by xvjiarui** [comment]
Sorry for the late reply. 
Do other models have the same issue? For example, PSPNet?
